modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -31,6 +45,41 @@ def find_bgs_claim!(claim_id:)
 
           raise
         end
+
+        def find_or_create_ews(bgs_claim)
+          ews = ClaimsApi::EvidenceWaiverSubmission.find_by(claim_id: bgs_claim[:bnft_claim_dto][:bnft_claim_id])
+
+          unless ews&.status&.in?(%w[submitted pending])
+            attributes = {
+              status: ClaimsApi::EvidenceWaiverSubmission::PENDING,
+              auth_headers: auth_headers,
+              cid: source_cid
+            }
+
+            ClaimsApi::EvidenceWaiverSubmission.create!(attributes)
+          end
+        end
+
+        def attempt_background_submit(bgs_claim, ews)
+          response = bgs_service.benefit_claims.update_bnft_claim(claim: bgs_claim)
+
+          if response[:bnft_claim_dto].nil?
+            ClaimsApi::EwsUpdater.perform_async(bgs_claim, target_veteran, ews)
+          else
+            ews[:status] = ClaimsApi::EvidenceWaiverSubmission::UPDATED
+
+            ClaimsApi::Logger.log('ews', claim_id: bgs_claim.dig[:bnft_claim_dto][:bnft_claim_id],
```

Comment:
invalid format. this will break. should be `bgs_claim.dig(:bnft_claim_dto, :bnft_claim_id)` for the claim id